### PR TITLE
Feature concrete class webcam

### PR DESCRIPTION
**Pull request by Jesperobb**
First start of the webcam class that uses the Picamera.

### CODE REVIEW

File: module/pi_cam.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+class PiCam:
+    """
+    Picam class
+    Default resolution = 1280 by 720
+    """
+    def __init__(self):
+        self.camera = PiCamera()
+        self.camera.resolution = (1280, 720)
+        time.sleep(2)
```
Comment (streefje1996) [inline module/pi_cam.py]: is this sleep here necessary

File: module/pi_cam.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    #    while self.comm.has_data():
+    #        frame = self.comm.get_data()
+
+    #       if frame.request:
+    #            continue
+
+    #        values = frame.get_data()
+
+    # def stop(self):
+    #    self.comm.stop()
```
Comment (streefje1996) [inline module/pi_cam.py]: commented code needs to be removed

File: module/pi_cam.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        self.camera.capture(timestr)
+
+    def set_resolution(self, x, y):
+        """
+        Changes the resolution of the PiCamera
+        :param x: amount of pixels for x
+        :param y: amount of pixels for y
+        :return:
+        """
+        self.camera.resolution = (x, y)
```
Comment (streefje1996) [inline module/pi_cam.py]: maybe an idea to add an constructor with these parameters as well

**Review by streefje1996 — CHANGES_REQUESTED** [review]
Remove the commented code from the class file,
In general looks good

File: module/pi_cam.py (hunk 1 above)
Comment (Jesperobb) [reply module/pi_cam.py]: According to the documentation it does need a little bit of time to startup.

File: module/pi_cam.py (hunk 2 above)
Comment (Jesperobb) [reply module/pi_cam.py]: Done, totally forgot. Thanks

File: module/pi_cam.py (hunk 1 above)
Comment (streefje1996) [reply module/pi_cam.py]: alright 👍

**Review by streefje1996 — APPROVED** [review]
Good cleanup,
Looks tight.
Good job 👍

**Pull request merged**